package.json

```diff
@@ -20,9 +20,7 @@
     "pioneer/packages/*",
     "utils/api-scripts",
     "content-directory-schemas",
-    "query-node",
-    "query-node/generated/graphql-server",
-    "query-node/generated/indexer"
```

Comment:
There is a small issue related to this change.

Because `query-node/generated/indexer` is no longer a monorepo workspace, root monorepo `resolutions` are not taken into account when modules are installed there (by `codegen`)

This means that after `yarn workspace query-node-root build` we're going to have multiple instances of `@polkadot/api`, `@polkadot/types` etc. in the monorepo:
```
Multiple instances of @polkadot/types detected, ensure that there is only one package in your dependency tree.
	1.34.1	/.../query-node/generated/indexer/node_modules/@polkadot/types
	1.26.1	/.../node_modules/@polkadot/types

```

If we then run `yarn workspace query-node-root indexer:start` - the indexer will use a newer version of `@polkadot/types`/`@polkadot/api` and it will cause it to fail when decoding some of the blocks, ie.:
```
Error: createType(SignedBlock):: Struct: failed on 'block':: Struct: failed on 'extrinsics':: createType(ExtrinsicV4):: createType(Call):: Struct: failed on 'args':: u16: Input too large. Found input with 27 bits, expected 16, stack: Error: Error: createType(SignedBlock):: Struct: failed on 'block':: Struct: failed on 'extrinsics':: createType(ExtrinsicV4):: createType(Call):: Struct: failed on 'args':: u16: Input too large. Found input with 27 bits, expected 16
```

I managed to resolve this issue locally by adding back ` "query-node/generated/*"` to root monorepo `package.json` `workspaces`:
```
  "workspaces": [
    /* ... */
    "query-node/generated/*"
   ]
```

Reply:
I did come across that at one point, and I remember distinctly the same error. And yes it was after removing them from the workspace. I tried to re create it several times later but couldn't, and I didn't see the same error looking through the docker logs of the. I'm pretty sure the version of the hydra-cli v0.0.17 is now selecting v1.26.1 the same version.

Let me test change it back and push and update and see if it works out ok.

Reply:
oh just spotted it you were correct
https://github.com/Joystream/joystream/pull/1639/checks?check_run_id=1345727724#step:9:671